mysql/src/main/scala/org/scalarelational/mysql/MysqlDatastore.scala

```diff
@@ -0,0 +1,85 @@
+package org.scalarelational.mysql
+
+;
+
+import javax.sql.DataSource
+
+
+import com.mysql.jdbc.jdbc2.optional.{MysqlDataSource}
+import org.powerscala.log.Logging
+import org.powerscala.property.Property
+import org.scalarelational.model._
+
+/**
+ * @author Matt Hicks <matt@outr.com>
+ */
+
+trait MysqlConnectionMode {
+  def url: String
+}
+
+object MysqlConnectionMode {
+  def apply(connectionURL: String) = new MysqlConnectionMode {
+    override def url = connectionURL
+  }
+}
+
+
+abstract class MysqlDatastore private() extends SQLDatastore with Logging {
+
+  override def DefaultVarCharLength = 100
```

Comment:
100 is definitely too short. From what I've seen 255 is most common as a default. I would leave it at 65k because this will capture most strings and reduce the need to increase the column size in many situations.